src/main/webapp/WEB-INF/views/cars/add.jsp

```diff
@@ -0,0 +1,15 @@
+<%@ page contentType="text/html;charset=UTF-8" language="java" %>
+<html>
+<head>
+    <title>Add drivers to car</title>
+</head>
+<body>
+<h1>Please enter IDs of car and driver to connect</h1>
+
+<form method="post" action="${pageContext.request.contextPath}/cars/add">
+    Car id <input type="number" name="car_id"><br>
+    Driver id <input type="number" name="driver_id"><br>
```

Comment:
```suggestion
    Car id <input type="number" name="car_id" required><br>
    Driver id <input type="number" name="driver_id" required><br>
```
**OPTIONAL**: read about this parameter, add where needed

Reply:
done